docs/api.md

```diff
@@ -81,156 +80,7 @@ These are the API group versions which are currently available when using Antrea
-   ignore these legacy resources: updates to the legacy resources (including
-   deletions) are not applied to the corresponding new resource any more, and
-   changes to the new resources are now possible (they will not be overwritten
-   by the controller). As an example, the command below will annotate *all* ANPs
-   in the current Namespace with `"crd.antrea.io/stop-mirror"`.
-
-   ```bash
-   kubectl annotate lanp.security.antrea.tanzu.vmware.com --all crd.antrea.io/stop-mirror=''
-   ```
-
-5. Check that none of the new Custom Resources still have the
-   `"crd.antrea.io/managed-by": "crdmirroring-controller"` annotation. Running
-   the same command as before should return an empty list:
-
-   ```bash
-   kubectl get anp.crd.antrea.io -o jsonpath='{range .items[?(@.metadata.annotations.crd\.antrea\.io/managed-by=="crdmirroring-controller")]}{.metadata.name}{"\n"}{end}'
-   ```
-
-   If you remove the filter, all your ANPs should still exist:
-
-   ```bash
-   kubectl get anp.crd.antrea.io -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}'
-   ```
-
-6. Safely delete all legacy CRDs previously managed by the application. As an
-   example, the command below will delete *all* legacy ANPs in the current
-   Namespace:
-
-   ```bash
-   kubectl delete lanp.security.antrea.tanzu.vmware.com
-   ```
-
-   Once again, all new ANPs should still exist, which can be confirmed with:
-
-   ```bash
-   kubectl get anp.crd.antrea.io -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}'
-   ```
-
-7. Start the new version of the application, which uses the new CRDs. All
-   mirrored Custom Resources should be available for the application to access.
-
-8. At this stage, if all applications have been updated, legacy CRD mirroring
-   can be disabled in the Antrea Controller configuration.
-
-Note that for CRDs which are "owned" by Antrea, `AntreaAgentInfo` and
-`AntreaControllerInfo`, resources are automatically created by the Antrea
-components using both API versions.
-
-### Deleting legacy Kubernetes resources after an upgrade
-
-After a successful upgrade from Antrea < v1.6 to Antrea >= v1.6, you may want to
-manually clean up legacy Kubernetes resources which were created by an old
-Antrea version but are no longer needed. Note that keeping these resource will
-not impact any Antrea functions.
-
-To delete these legacy resources (CRDs and webhooks), run:
-
-```bash
-kubectl get crds -o=name --no-headers=true | grep "antrea\.tanzu\.vmware\.com" | xargs -r kubectl delete
-kubectl get mutatingwebhookconfigurations -o=name --no-headers=true | grep "antrea\.tanzu\.vmware\.com" | xargs -r kubectl delete
-kubectl get validatingwebhookconfigurations -o=name --no-headers=true | grep "antrea\.tanzu\.vmware\.com" | xargs -r kubectl delete
-```
+[^1]: The v1alpha2 version of the `ClusterGroup` CRD is no longer served by the
+      apiserver in v1.12 and is completely removed in v2.0.0.
```

Comment:
I don't think this is correct. We reverted this change later on (see https://github.com/antrea-io/antrea/pull/5277), pending the addition of the CLI storage conversion tool.

Reply:
Should I just remove this comment and keep following?
```
| CRD | CRD version | Introduced in | Deprecated in | Removed in |
|---|---|---|---|---|
| `ClusterGroup` | v1alpha2 | v1.0.0 | v1.1.0 | v2.0.0 |
```

Reply:
I think so.

Reply:
Done